jemalloc-sys/build.rs

```diff
@@ -111,10 +111,16 @@ fn main() {
             .iter()
             .any(|i| target.contains(i))
     {
-        warning!(
-            "Unprefixed `malloc` requested on unsupported platform `{}` => using prefixed `malloc`",
-            target
-        );
+        // Apple targets don't support unprefixed, but they do support
+        // overriding (if you do the `zone_register` trick), so no need to
+        // warn there.
+        let override_ = env::var("CARGO_FEATURE_OVERRIDE").is_ok();
+        if !target.contains("apple") && !override_ {
```

Comment:
Should be `!target.contains("apple") || !override_`.

Reply:
True, the intention is:
- Apple target, request unprefixed -> warn
- Apple target, request override -> no warning
- Android target, request unprefixed -> warn
- Android target, request override -> warn